src/routes/safe/store/actions/createTransaction.js

```diff
@@ -99,38 +101,54 @@ const createTransaction = ({
 
   let txHash
   let tx
+  const txArgs = {
```

Comment:
I moved this to a generic place. to avoid bugs if you forget to modify the TXs arguments in both places (exec and saveToHistory). that produces duplicated TXs in the Transactions tab.